### DISCUSSION TITLE
GitHub Issue Template Tidak Compatible Dengan GitHub Mobile

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1594] [Q&A] [answered]
Mengapa klik general jadi seperti ini di apk 
![Screenshot_2023-10-19-06-28-45-90_c31b32364ce19ca8fcd150a417ecce58.jpg](https://github.com/realodix/AdBlockID/assets/114223791/09594ef1-8bc3-404c-bd76-291e6c48d61d)

**Comment by realodix** [comment]
Pakai aplikasi Github ya?

Pilih saja aplikasi Github lagi, atau masuk ke browser

**Comment by Jokopentil** [comment]
![Screenshot_2023-10-19-12-56-10-73_c31b32364ce19ca8fcd150a417ecce58.jpg](https://github.com/realodix/AdBlockID/assets/114223791/2320f3cb-fc3f-4c3f-84d3-bbfae88b7489)

Tampilan seperti ini GK bisa terhubung apk github
Padahal lebih mudah pakai aplikasi daripada browser
Karena browser capek login jika sandi terhapus atau lupa

**Reply by Jokopentil** [reply]
![Screenshot_2023-10-19-12-56-10-73_c31b32364ce19ca8fcd150a417ecce58.jpg](https://github.com/realodix/AdBlockID/assets/114223791/f531f075-9e45-4c9b-bbc4-29ac17cb5455)

**Reply by realodix** [reply, marked as answer]
> Tampilan seperti ini GK bisa terhubung apk github
> Padahal lebih mudah pakai aplikasi daripada browser
> Karena browser capek login jika sandi terhapus atau lupa

Langsung laporkan saja ke 
- https://github.com/orgs/community/discussions/categories/projects-and-issues?discussions_q=category%3A%22Projects+and+Issues%22
- https://github.com/orgs/community/discussions/categories/mobile?discussions_q=category%3AMobile

---

Ini masih beta sih memang. Lucu juga, mereka yang nyaranin, malah belum compatible dengan aplikasinya sendiri.

![image](https://github.com/realodix/AdBlockID/assets/1314456/5a91fcf1-9ef4-4394-afe9-6b35e39ae80f)
